CafeTime/CafeTime/AuthManager.swift

```diff
@@ -136,7 +141,8 @@ class AuthManager {
         if FIRAuth.auth()?.currentUser != nil {
             do {
                 try FIRAuth.auth()?.signOut()
-                GlobalSaver.shared.clearUser()
+                let realm = RealmManager()
```

Comment:
Better to add  something like AuthDAO, where you will keep all methods that are working with Realm. and only in this file you will create realm instance